### ISSUE TITLE
Add support for anki 2.1

### ISSUE THREAD

**Issue by Mandascoop** [NinKenDo64/Jieba-Morph#2]
Hello,

Since this is such a great tool, and with the release of anki 2.1, i thought it could be nice if the tool could be used with anki 2.1. Since the support for chinese is only in this version that isnt compatible as of yet with anki 2.1.

Many thanks in advance

**Comment by NinKenDo64** [comment]
Hey Mandascoop. Since this issue hasn't much progressed I figured I'd update you on this.

So there is a branch of MorphMan that is attempting to build 2.1 support, but it doesn't seem to work for me. I was waiting for that project to reach a feature parity or at least actually work, before I began work on this.

Unfortunately that hasn't happened.

I have plans to go ahead and do it myself and in the process hopefully uncoupled Jieba as an optional dependency for those who want Chinese support (similar to how one uses an external copy of MeCab from the 'Japanese Support' add-on in order to do Japanese sentence parsing).

At the moment though I have literally no time, as I'm studying and working, and then obviously working on my languages. In the meantime, one can sync up a portable installation of Anki2.0 through AnkiWeb and reorder cards as needed using MorphMan there, and then reopen 2.1 and sync those changes down.

Not perfect, and you lose some functionality, but for card ordering it works pretty alright.

Thanks and sorry about not having time to work on it at the moment. As an Anki 2.1 user myself, it's definitely my first priority for my next project.
